packages/connect-common/files/coins.json

```diff
@@ -1068,54 +1090,6 @@
             "xpub_magic_segwit_native": 78792518,
             "xpub_magic_segwit_p2sh": 77429938
         },
-        {
-            "address_type": 38,
-            "address_type_p2sh": 62,
-            "bech32_prefix": "game",
-            "blockchain_link": {
```

Comment:
This object is being deleted by running the script `scripts/update-coins.sh`. I assume this is not in the firmware anymore and therefore it is deleted. @mroz22

Reply:
yes, that should be ok